README.md

```diff
@@ -143,7 +143,7 @@ Here OpenShift is used as the Kubernetes platform, you can use the [oc-login act
 - name: Knative Service Deploy
   id: kn_service_deploy
   uses: redhat-actions/kn-service-deploy@v1
-  with: 
+  with:
     service_name: fruits-app
```

Comment:
since we aren't using the fruits app anymore, should we replace the example name with something more generic?

Reply:
Yes, we should. Let me update.